std/node/_fs/_fs_stat_test.ts

```diff
@@ -17,9 +17,9 @@ export function assertStats(actual: Stats, expected: Deno.FileInfo) {
   assertEquals(actual.atime?.getTime(), expected.atime?.getTime());
   assertEquals(actual.mtime?.getTime(), expected.mtime?.getTime());
   assertEquals(actual.birthtime?.getTime(), expected.birthtime?.getTime());
-  assertEquals(actual.atimeMs, expected.atime?.getTime());
-  assertEquals(actual.mtimeMs, expected.mtime?.getTime());
-  assertEquals(actual.birthtimeMs, expected.birthtime?.getTime());
+  assertEquals(actual.atimeMs ?? undefined, expected.atime?.getTime());
+  assertEquals(actual.mtimeMs ?? undefined, expected.mtime?.getTime());
+  assertEquals(actual.birthtimeMs ?? undefined, expected.birthtime?.getTime());
```

Comment:
Changes in this file seem unrelated, please revert

Reply:
Ok, I've moved them to #9311.